vendor/dell/z9264f/etc/opx/acl/nas_detail_list.xml

```diff
@@ -507,5 +508,6 @@
              </value>
          </match>
      </entry>
+
```

Comment:
Please delete this line to make the spacing consistent with the files for the other platforms (sorry for the nitpick)